x-pack/plugins/security_solution/scripts/endpoint/action_responder/utils.ts

```diff
@@ -144,6 +147,40 @@ export const sendEndpointActionResponse = async (
     }
   }
 
+  // For `get-file`, upload a file to ES
+  if (action.command === 'get-file' && !endpointResponse.error) {
+    // Add the file's metadata
+    const fileMeta = await esClient.index<UploadedFile>({
+      index: FILE_STORAGE_METADATA_INDEX,
+      id: `${action.id}.${action.hosts[0]}`,
+      body: {
+        file: {
+          created: new Date().toISOString(),
+          extension: 'zip',
+          path: '/some/path/bad_file.txt',
+          type: 'file',
+          size: 221,
+          name: 'bad_file.txt.zip',
+          mime_type: 'application/zip',
+          Status: 'READY',
+          ChunkSize: 4194304,
+        },
+      },
+      refresh: 'wait_for',
+    });
+
+    await esClient.index({
+      index: FILE_STORAGE_DATA_INDEX,
+      id: `${fileMeta._id}.0`,
```

Comment:
@pzl - I think this is right. I did not have a file with multiple chunks, but looking at the data you sent me, I assumed that each chunk's id would be comprised of the File's meta document `_id` + `.<chunk_number>`.

Reply:
precisely correct